tests/api.spec.ts

```diff
@@ -465,4 +466,16 @@ describe('Base64 encoded operations', () => {
         ],
       });
     });
+
+  describe('it auto-sets base64 encoding if needed', () => {
+    console.log(baseURL);
+    expect(setBase64Encoding({ baseURL, sessionToken, source, base64Encoding: false })).toBe(false);
+    expect(setBase64Encoding({ baseURL, sessionToken, source, base64Encoding: true })).toBe(true);
+    expect(
+      setBase64Encoding({ baseURL: 'https://deeproxy.dev.eu.snyk.io', sessionToken, source, base64Encoding: false }),
+    ).toBe(true);
+    expect(
+      setBase64Encoding({ baseURL: 'https://deeproxy.snyk.io', sessionToken, source, base64Encoding: false }),
+    ).toBe(false);
+  });
```

Comment:
suggestion: maybe add an expect for dev.snyk.io for clarity?

Reply:
`dev.snyk.io` is set as `baseURL` from `./constants/base` - happy to add this here explicitly though